mobile/client.go

```diff
@@ -92,6 +95,19 @@ func (ps *Props) GetInt(key string) (val int, err error) {
 	return
 }
 
+func (ps *Props) GetInt64(key string) (val int64, err error) {
+	if x, found := ps.m[key]; found {
+		if i64, ok := x.(int64); ok {
+			val = i64
+		} else if i, ok := x.(int); ok {
+			val = int64(i)
+		} else {
```

Comment:
It could also be a float64, because Props may be unmarshaled from JSON, and there is no number type information (Go uses float64 in such a case). The other number getters might also have to be updated, either to handle int64 (GetFloat) or to report a more accurate error message (GetInt might not be able to represent the number).